examples/testing.md

```diff
@@ -0,0 +1,59 @@
+## Testing
+
+Testing in software development is a process that aims to evaluate the functionality of an application with an intent to find whether the code met the specified requirements or not as well as identifying the problems to ensure that the product has expected quality.
+
+## Automated tests
+
+Automated tests follow the process of testing the software using an automation tool to find the defects. In this process, programmers execute the test scripts and generate the test results automatically by using automation tools.
+
+## Tests in V
+
+In V, all test files have to be named with following format: `*_test.v` and the functions should start with `test_`.
```

Comment:
I think just add the word `the` before `following`
> In V, all test files have to be named with the following format: `*_test.v` and the functions should start with `test_`.

Reply:
sounds good! Adding now.